tests/sentry/event_manager/test_event_manager_grouping.py

```diff
@@ -137,60 +136,48 @@ def test_updates_group_metadata(self):
     def test_auto_updates_grouping_config(self):
         self.project.update_option("sentry:grouping_config", LEGACY_CONFIG)
 
-        with override_settings(SENTRY_GROUPING_AUTO_UPDATE_ENABLED=False):
```

Comment:
This test change can be more easily seen by hiding white spaces. This screenshot should save you having to reload the page:
<img width="724" alt="image" src="https://github.com/user-attachments/assets/3a4f5202-c054-4192-ada2-999de1bb3028">